docs/how-to/code/use-django-extension/override_example/rockcraft.yaml

```diff
@@ -0,0 +1,29 @@
+name: example-django
+summary: A Django application
+description: A rock packing a Django application
+version: "0.1"
+base: ubuntu@22.04
+license: Apache-2.0
+
+extensions:
+  - django-framework
```

Comment:
```suggestion
  - django-framework
  
platforms:
  amd64:
  arm64:
```

This will fix the spread test failure.

Reply:
thank you for that, should be fixed now

Reply:
Also needed to do it for the other example

Reply:
I think the failure now is not related to this change

Reply:
I concur. Thanks for the update!